docs/buildstrategies.md

```diff
@@ -6,26 +6,26 @@ SPDX-License-Identifier: Apache-2.0
 
 # BuildStrategies
 
-- [Overview](#overview)
-- [Available ClusterBuildStrategies](#available-clusterbuildstrategies)
-- [Available BuildStrategies](#available-buildstrategies)
-- [Buildah](#buildah)
-  - [Installing Buildah Strategy](#installing-buildah-strategy)
-- [Buildpacks v3](#buildpacks-v3)
-  - [Installing Buildpacks v3 Strategy](#installing-buildpacks-v3-strategy)
-  - [Try it](#try-it)
-- [Kaniko](#kaniko)
-  - [Installing Kaniko Strategy](#installing-kaniko-strategy)
-- [ko](#ko)
-  - [Installing ko Strategy](#installing-ko-strategy)
-- [Source to Image](#source-to-image)
-  - [Installing Source to Image Strategy](#installing-source-to-image-strategy)
-  - [Build Steps](#build-steps)
-- [Steps Resource Definition](#steps-resource-definition)
-  - [Strategies with different resources](#strategies-with-different-resources)
-  - [How does Tekton Pipelines handle resources](#how-does-tekton-pipelines-handle-resources)
-  - [Examples of Tekton resources management](#examples-of-tekton-resources-management)
-- [Annotations](#annotations)
+- [BuildStrategies](#buildstrategies)
```

Comment:
GitHub generates its own table of contents now, we might consider not writing our own anymore (doesn't have to be in this PR, just something to consider)